### PR TITLE
Scan reveals invisibility (60 seconds cooldown/team)

### PR DESCRIPTION
**Pull request by Zarnotox**
That didn't take too long to implement, right?

### CODE REVIEW

**Comment by Haganeko** [comment]
Not once you got started XD

File: game/scripts/vscripts/modifiers/modifier_scan_true_sight.lua [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+if IsServer() then
+
+    function modifier_scan_true_sight_thinker:OnCreated( event )
+
+        self.debuff_interval = 0.4
+        self.debuff_duration = 0.5
+        self.radius = SCAN_REVEAL_RADIUS
+
+        self:StartIntervalThink(self.debuff_interval)
```
Comment (Trildar) [inline game/scripts/vscripts/modifiers/modifier_scan_true_sight.lua]: Though it is called a modifier thinker, you could just write this as an aura. Though I don't know if it's necessarily worth the effort now.

Comment (Zarnotox) [reply game/scripts/vscripts/modifiers/modifier_scan_true_sight.lua]: I can still do that, have to look up how auras work first though

**Comment by chrisinajar** [comment]
Closes: #1519

**Pull request merged**